.github/workflows/dev_cpu_linux_riscv.yml

```diff
@@ -0,0 +1,269 @@
+            -DENABLE_TESTS=ON \
+            -DENABLE_INTEL_CPU=ON \
+            -DENABLE_INTEL_GPU=OFF \
+            -DENABLE_INTEL_NPU=OFF \
+            -DENABLE_SAMPLES=OFF \
+            -DCMAKE_TOOLCHAIN_FILE=${OPENVINO_REPO}/cmake/toolchains/riscv64-071-xuantie-gnu.toolchain.cmake \
+            -DRISCV_TOOLCHAIN_ROOT=${XUANTIE_DIR}/${XUANTIE_TAR_NAME} \
+            -DENABLE_PYTHON=OFF \
+            -DENABLE_PYTHON_PACKAGING=ON \
+            -DENABLE_WHEEL=OFF \
+            -DENABLE_STRICT_DEPENDENCIES=OFF \
+            -DCMAKE_VERBOSE_MAKEFILE=ON \
+            -DCPACK_GENERATOR=TGZ \
+            -DCMAKE_COMPILE_WARNING_AS_ERROR=OFF \
+            -DCMAKE_CXX_COMPILER_LAUNCHER=${{ env.CMAKE_CXX_COMPILER_LAUNCHER }} \
+            -DCMAKE_C_COMPILER_LAUNCHER=${{ env.CMAKE_C_COMPILER_LAUNCHER }} \
+            -S ${OPENVINO_REPO} \
+            -B ${BUILD_DIR}
+
+      - name: Cmake build - OpenVINO
+        run: cmake --build ${BUILD_DIR} --parallel $(nproc) --config ${{ env.CMAKE_BUILD_TYPE }}
+
+      - name: Show ccache stats
+        run: ccache --show-stats
+
+      - name: Cmake install - OpenVINO
+        run: |
+          cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR} -P ${BUILD_DIR}/cmake_install.cmake
+          cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_TEST_DIR} -DCOMPONENT=tests -P ${BUILD_DIR}/cmake_install.cmake
+
+      - name: Pack Artifacts
+        run: |
+
+          pushd ${INSTALL_DIR}
+            tar -czvf ${BUILD_DIR}/openvino_package.tar.gz *
+          popd
+
+          pushd ${INSTALL_TEST_DIR}
+            tar -czvf ${BUILD_DIR}/openvino_tests.tar.gz *
+          popd
+
+      # #
+      # # Upload build artifacts
+      # #
+
+      - name: Upload openvino package
+        if: ${{ always() }}
+        uses: actions/upload-artifact@0b2256b8c012f0828dc542b3febcab082c67f72b # v4.3.4
+        with:
+          name: openvino_package
+          path: ${{ env.BUILD_DIR }}/openvino_package.tar.gz
+          if-no-files-found: 'error'
+
+      - name: Upload openvino tests package
+        if: ${{ always() }}
+        uses: actions/upload-artifact@0b2256b8c012f0828dc542b3febcab082c67f72b # v4.3.4
+        with:
+          name: openvino_tests
+          path: ${{ env.BUILD_DIR }}/openvino_tests.tar.gz
+          if-no-files-found: 'error'
+    
+      - name: Clean ccache
+        run: ccache --cleanup
+
+  CPU_Functional_Tests:
+    name: CPU functional tests
+    needs: [ Docker, Build, Smart_CI ]
+    timeout-minutes: 30
+    runs-on: aks-linux-4-cores-16gb
+    container:
+      image: ${{ fromJSON(needs.docker.outputs.images).ov_build.ubuntu_22_04_riscv }}
+      volumes:
+        - /mount:/mount
+    defaults:
+      run:
+        shell: bash
+    env:
+      INSTALL_DIR: ${{ github.workspace }}/install
+      INSTALL_TEST_DIR: ${{ github.workspace }}/install/tests
+      GTEST_FILTER: ${{ github.event_name == 'workflow_dispatch' && inputs.testFilter || '*smoke_AdaPoolAvg4DLayoutTest*' }}
```

Comment:
Do we validate only `smoke_AdaPoolAvg4DLayoutTest` tests in this job?
As far as I remember, we discussed offline a few months ago about testing FullyConnected and Eltwise layers at least. Because:
- Using emulator - is slow
- Xuantie qemu has memory leak during execution. 
- Only these ops have optimized impls from SHL.

So we cannot launch all tests, for sure. But let's test only updated layers at least

Reply:
Usw manual launch. As we discussed

Reply:
Use it as an example for a future https://github.com/openvinotoolkit/openvino/actions/runs/12467513502/job/34798900269